app/views/common/form/_new_password_input.html.slim

```diff
@@ -1,10 +1,10 @@
 = f.dsfr_input_group :password, class: "fr-password fr-mt-2w", data: {component: "js_dsfr_new_password"}
-  = f.dsfr_label_with_hint :password, class: "fr-label fr-password__label", hint: "Choisissez votre mot de passe"
+  = f.dsfr_label_with_hint :password, class: "fr-label fr-password__label", hint: defined?(hint) ? hint : nil, label: defined?(password_label) ? password_label : nil
```

Comment:
je crois qu'on pourrait remplacer `defined?(hint) ? hint : nil` par `locals[:hint]`. C'est peut-être aussi une bonne occasion d'explorer les nouvelles définition de locals de partials proposées par rails 7.1 https://blog.kiprosh.com/allow-template-to-set-strict-locals/

Reply:
Très bonne proposition, c’est vrai que c’est beaucoup mieux comme ça